docs/www/acls.md

```diff
@@ -56,61 +62,160 @@ redpanda:
-  --tls-cert <path to PEM-formatted cert file> \
-  --tls-truststore <path to PEM-formatted truststore file>
+```bash
+ rpk topic create test-topic \
+--tls-key <path to PEM-formatted key file> \
+--tls-cert <path to PEM-formatted cert file> \
+--tls-truststore <path to PEM-formatted truststore file>
+```
 
+```bash
 Created topic 'test-topic'.
-You may check its config with
+```
 
+You may check its config with:
+
+```bash
 rpk topic describe 'test-topic'
 ```
 
-> `rpk` defaults to connecting to `localhost:9092`. If you're connecting to a remote broker, you will need to pass `--brokers <node IP>:<kafka API port>`
+**_Note_** - `rpk` defaults to connecting to `localhost:9092`. If you're connecting to a remote broker, you will need to pass `--brokers <node IP>:<kafka API port>`
 
-> For an in-depth guide about enabling TLS and mutual TLS in redpanda check out [our guide on the subject](https://vectorized.io/blog/tls-config/).
+For an in-depth guide about enabling TLS and mutual TLS in redpanda check out [our guide on the subject](https://vectorized.io/blog/tls-config/).
 
 ### SASL/ SCRAM
 
 Redpanda also supports client authentication via SASL/SCRAM (that is, the Simple Authentication and Security Layer protocol, using the Salted Challenge Response Authentication Mechanism), which is based on usernames & passwords.
 
-To enable it, set `redpanda.enable_sasl` to `true` in the configuration file, as well as list at least one "superuser" which after created will have permissions for all operations on the clusters.
+To enable it, set `redpanda.enable_sasl` to `true` in the configuration file, as well as list at least one "superuser" which after being created will have permissions for all operations on the clusters.
+
+You config must look something like this:
 
 ```yaml
 redpanda
   
   enable_sasl: true
   superusers:
-  - admin
+    - username: admin
   # The rest of the config...
 ```
 
+<tabs>
+  <tab id="Local Redpanda">
+  You can find the configuration file here
+
+  ```bash
+  /etc/redpanda/redpanda.yaml
+  ```
+
+After you change it's value, restart Redpanda service for changes to be in effect. 
+
+  </tab>
+  <tab id="Docker Container">
+  To access the files inside the container, first you have to enter the container.
+
+  You can do that by running:
+
+```bash
+  docker exec -it <name-of-container> bash
+```
+
+Change <name-of-container> for your container and execute the command. 
+Then go to the same directory as a local redpanda config:
+
+```bash
+  /etc/redpanda/redpanda.yaml
+```
+
+After you finish it, restart the container for changes to be in effect. 
+
+  </tab>
+  <tab id="Kubernetes">
+  For Kubernetes, things are a little different. You can change your configuration file with the YAML for the cluster.
+
+  If you got our [one-cluster sample](https://github.com/vectorizedio/redpanda/blob/dev/src/go/k8s/config/samples/one_node_cluster.yaml) you should add these values to enable different flags.
```

Comment:
- Correct grammar
- "should" indicates a possibility whereas this is the required way to set SASL in K8s.
- Specify the fields to change

```suggestion
  If you're using our [one-cluster sample](https://github.com/vectorizedio/redpanda/blob/dev/src/go/k8s/config/samples/one_node_cluster.yaml), specify the `redpanda.enable_sasl` and `superuser` values in the cluster spec YAML file.
```